### PR TITLE
feat: Configure GitHub action to ingest docs site on PR merge

### PR DESCRIPTION
**Pull request by mrderyk**
**CHANGES**
- Uncomment job to ingest docs site upon merging changes to www directory

### CODE REVIEW

File: .github/workflows/ingest-docs.yml [hunk 1]
```diff
@@ -1,60 +1,48 @@
...
-#             pos_regex: [\".*vectara.com/docs.*\"]
-#             neg_regex: [\".*vectara.com/docs/rest-api/.*\"]
-#             extensions_to_ignore: [\"php\", \"java\", \"py\", \"js\"]
-#             docs_system: docusaurus" >> ingest.yaml
+          docs_crawler:
+            base_urls: [\"https://docs.vectara.com/docs\"]
+            pos_regex: [\".*vectara.com/docs.*\"]
+            neg_regex: [\".*vectara.com/docs/rest-api/.*\"]
+            extensions_to_ignore: [\"php\", \"java\", \"py\", \"js\"]
+            docs_system: docusaurus" >> ingest.yaml
```
Comment (cjcenizal) [inline .github/workflows/ingest-docs.yml:36]: This double quotation mark looks odd. Is that intentional?

File: .github/workflows/ingest-docs.yml [hunk 2]
```diff
@@ -1,60 +1,48 @@
...
-#           cat secrets.toml
-#       - name: Run the bash script
-#         working-directory: ./vectara-ingest
-#         run: |
-#           bash run.sh ingest.yaml default
-#       - name: Check the logs
-#         working-directory: ./vectara-ingest
-#         run: docker logs -f vingest
+          echo "[default]
+          api_key=\"${{secrets.VECTARA_DOCS_API_KEY}}\"" >> secrets.toml
```
Comment (cjcenizal) [inline .github/workflows/ingest-docs.yml:39]: I guess it must be because there's another hanging double quotation mark here, right?

**Review by cjcenizal — APPROVED** [review]
🤯 This looks awesome! @ofermend Check it out, we're using Vectara Ingest!

File: .github/workflows/ingest-docs.yml (hunk 2 above)
Comment (mrderyk) [reply .github/workflows/ingest-docs.yml:39]: Yup that is correct! We're basically printing this whole line to the new file.

**Pull request merged**